sdk/appcenter-distribute/src/test/java/com/microsoft/appcenter/distribute/DistributeUtilsTest.java

```diff
@@ -5,13 +5,120 @@
 
 package com.microsoft.appcenter.distribute;
 
+import com.microsoft.appcenter.utils.storage.SharedPreferencesManager;
+
+import org.json.JSONException;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Rule;
 import org.junit.Test;
+import org.powermock.core.classloader.annotations.PrepareForTest;
+import org.powermock.modules.junit4.rule.PowerMockRule;
+
+import static com.microsoft.appcenter.distribute.DistributeConstants.PREFERENCE_KEY_RELEASE_DETAILS;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNull;
+import static org.mockito.Matchers.anyString;
+import static org.mockito.Matchers.eq;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.never;
+import static org.mockito.Mockito.when;
+import static org.powermock.api.mockito.PowerMockito.mockStatic;
+import static org.powermock.api.mockito.PowerMockito.verifyStatic;
 
+@PrepareForTest({DistributeUtils.class, SharedPreferencesManager.class, ReleaseDetails.class})
 public class DistributeUtilsTest {
 
+    @Rule
+    public PowerMockRule mRule = new PowerMockRule();
+
+    @Before
+    public void setUp() {
+        mockStatic(SharedPreferencesManager.class);
+        mockStatic(ReleaseDetails.class);
+    }
+
     @Test
     public void init() {
         new DistributeUtils();
         new DistributeConstants();
     }
+
+    @Test
+    public void getNotificationId() {
+
+        /* Coverage fix. */
+        int notificationId = DistributeUtils.getNotificationId();
```

Comment:
If it's never called anywhere, why do we even need the method?

Reply:
It's called by `notificationManager`

Reply:
I'm afraid I'd have to return it. Test with calling this method, when executed from console, fail with an NPE, however they pass fine from Android Studio, even if I execute the full batch.
Unfortunately I don't think that's something we should invest more time into give the circumstances with the deadlines.